ProjectSidewalk/SidewalkWebpage
# 3242: Redesigned explore mission screens

Resolves #3242

Implements redesigned explore mission screens.

##### Before/After screenshots (if applicable)
Before
<img width="1440" alt="Screenshot 2023-05-24 at 10 27 26 AM" src="https://github.com/ProjectSidewalk/SidewalkWebpage/assets/8168506/b2a58537-2014-4489-bb81-96285971860f">

After
<img width="1440" alt="Screenshot 2023-05-24 at 10 29 28 AM" src="https://github.com/ProjectSidewalk/SidewalkWebpage/assets/8168506/64642642-4982-48a8-bf36-135310fbb63f">


##### Testing instructions
1. Open an explore mission and see that the new mission screens are shown. (Note: the new mission screens are shown only if the tutorial is skipped).
2. See that mission screens for a randomly selected label are displayed.
3. Go through all the slides for the label type by clicking on the arrows. 
4. Make sure that the image and text content on slides match.
5. See that the 'Start Mission' button does not animate once you reach the end of the slides (this is intentional as we want users to see other label types as well).
6. Click on all the label types and repeat steps 4 and 5.
7. Ensure that there is no change in UI performance and also all UI elements are interactive (because of event handler management).
8. After checking all label types, click on Start Mission button and ensure that you can continue with explore mission as usual.
9. Also ensure that there are no regressions in validate mission screens.

##### Things to check before submitting the PR 

- [ ] I've tested on mobile (only needed for validation page). (NA)
